### PR TITLE
Added RedShelf

### LINKED ISSUE #5270: Add RedShelf
- [x] I have searched both open and closed [issues](https://github.com/2factorauth/twofactorauth/issues). The issue I'm creating is not a duplicate of an existing issue.
- [x] I have searched both open and closed [pull requests](https://github.com/2factorauth/twofactorauth/pulls). The issue I'm creating is not a duplicate of an existing pull request.

### Information about the site to be added that supports 2FA: ###
* RedShelf

* https://www.redshelf.com

* Does the site support 2FA? - `Yes`

* Documentation URL
* https://solve.redshelf.com/hc/en-us/articles/1260802590609-Using-Multi-Factor-Authentication-

### Information about what methods of 2FA the site supports: ###

* Software:
  - [x] TOTP (RFC-6238, Google Authenticator)
  - [ ] Proprietary (Authy, Duo, etc.)

* Hardware:
  - [ ] FIDO2/U2F security keys
  - [ ] Other

- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Information about the site's eligibility to be added to 2fa.directory: ###

- [x] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md).
- [x] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [x] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.

### Additional information: ###

This also works on brytewave.redshelf.com/ which is part of the same website. I'm not sure what the differences are from each of the website, but I assume they serve the same function.

### PR DESCRIPTION
**Pull request by ApCoder123**
Resolves #5270

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -457,6 +457,13 @@ websites:
     twitter: quizlet
     facebook: quizlet
 
+  - name: RedShelf
+    url: https://www.redshelf.com/
+    img: redshelf.png
+    tfa:
+      - totp
+    doc: https://solve.redshelf.com/hc/en-us/articles/1260802590609
+
```
Comment (Carlgo11) [inline _data/education.yml:468]:
```suggestion
    regions:
      - us

```

File: _data/education.yml [hunk 2]
```diff
@@ -457,6 +457,13 @@ websites:
     twitter: quizlet
     facebook: quizlet
 
+  - name: RedShelf
+    url: https://www.redshelf.com/
+    img: redshelf.png
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
    img: redshelf.svg
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Use this image instead:

**Comment by ApCoder123** [comment]
Okay, I have updated the branch with the requested changes.

**Review by Carlgo11 — APPROVED** [review]
Super. Thanks! 👍

**Review by phallobst — APPROVED** [review]

**Pull request merged**